includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -81,13 +83,56 @@ public function get_selector_conversion_mapping() {
 		];
 	}
 
+	/**
+	 * Convert picture element into image element or mark as px verified.
+	 *
+	 * @return void
+	 */
+	protected function process_picture_elements() {
+
+		$nodes = $this->dom->xpath->query( '//picture//img' );
```

Comment:
This could be made to select the direct descendant:

```suggestion
		$nodes = $this->dom->xpath->query( '//picture/img' );
```

Reply:
Or better to select `picture` elements that have `img` children:

```suggestion
		$nodes = $this->dom->xpath->query( '//picture[ img ]' );
```

Reply:
Or or no, the first is better, as then you can loop over the `img` elements and then to get the `picture` you just get the `parentNode`.